### PR TITLE
Feature/add logging

### PR DESCRIPTION
**Pull request by nlog14**
logging is set up

### CODE REVIEW

**Review by Kether180 — APPROVED** [review]

File: docker-compose.yml [hunk 1]
```diff
@@ -44,3 +44,35 @@ services:
       - ./local-volumes/grafana-data:/var/lib/grafana # To run locally uncomment this
     ports:
       - "3000:3000"
```
Comment (ChristofferTofteng) [inline docker-compose.yml:46]: I guess this should be commented out.

Comment (ChristofferTofteng) [reply docker-compose.yml:46]: Otherwise looks fine if it works, I'm not that strong in the logging department

Comment (nlog14) [reply docker-compose.yml:46]: this file isn't copied to the server so it doesn't get run remotely only locally

Comment (ChristofferTofteng) [reply docker-compose.yml:46]: Then no prob here

**Pull request merged**